crev-lib/src/proofdb.rs

```diff
@@ -957,47 +960,43 @@ impl ProofDB {
             .map(|review| review.package.id.version.clone())
     }
 
-    fn record_url_from_to_field(&mut self, date: &DateTime<Utc>, to: &crev_data::PubId) {
-        self.url_by_id_secondary
-            .entry(to.id.clone())
-            .or_insert_with(|| TimestampedUrl {
-                value: to.url.clone(),
-                date: *date,
-            });
-    }
-
-    fn record_url_from_from_field(&mut self, date: &DateTime<Utc>, from: &crev_data::PubId) {
+    /// Record mapping between a PubId and a URL it declares, and trust it's correct only if it's been fetched from the same URL
+    fn record_url_for_id(&mut self, date: &DateTime<Utc>, pubid: &crev_data::PubId, fetched_from: &FetchSource) {
```

Comment:
Previously we had much stronger "verified" criteria - it was verified only if it was reported and signed by the same ID.

Reply:
I don't know if I understand everything right, as I'm only quickly skimming, but it seems we should keep the self-reported and signed ID (best kind) as a primary, and then do some more heuristics on the secondary ones?

Reply:
Yeah, that makes sense

Reply:
fixed in 75391be4220638be1418be911cc8307d9fd5a0d8